src/App.tsx

```diff
@@ -150,8 +161,24 @@ function App() {
                     {jobLogs}
                 </div>
             )}
+            {
+                resultUrl && (
+                    <div>
+                        <iframe
```

Comment:
in the long run we could also install the viewer as component instead of doing this, but it's good for now